src/ceph-disk

```diff
@@ -711,7 +711,7 @@ def get_conf(cluster, variable):
                 'ceph-conf',
                 '--cluster={cluster}'.format(
                     cluster=cluster,
-                    ),
+                    ) if cluster != None else '',
```

Comment:
I've observed cluster being None in this function. I put my theory of why this happens in the pull request note.

This does work for me, however now I've done some more reading of ceph_conf.cc I think I'm merely getting away with a blank argument rather than it being processed. I'm going to make another version which removes the argument entirely. This does mean a larger diff unfortunately.